src/ReactFilestack.jsx

```diff
@@ -71,15 +73,16 @@ class ReactFilestack extends Component {
   };
 
   render () {
-    const { buttonClass, buttonText, link } = this.props;
+    const { buttonClass, buttonText, link, children } = this.props;
     const Tag = link ? 'a' : 'button';
     return (
       <Tag
         name="filestack"
         onClick={this.onClickPick}
         className={buttonClass}
       >
-        {buttonText}
+        {!children && buttonText}
```

Comment:
Yes this is exactly what I want to do. Actually I would prefer to replace `buttonText` by `children` but I don't want to break compatibility.